templates/install-wp-tests.sh

```diff
@@ -116,7 +116,13 @@ install_db() {
 			EXTRA=" --host=$DB_HOSTNAME --protocol=tcp"
 		fi
 	fi
-
+	
+	# drop database if exists
+    RESULT=`mysqlshow --user="$DB_USER" --password="$DB_PASS"$EXTRA --port $DB_SOCK_OR_PORT -h $DB_HOSTNAME $DB_NAME| grep -v Wildcard | grep -o $DB_NAME`
```

Comment:
Why does `--port $DB_SOCK_OR_PORT -h $DB_HOSTNAME` need to be supplied if we're already including `$EXTRA` ?